Tests are broken

### Description
The tests are broken on master. #29 broke test_storage_ignores_duplicate_ids and #30 broke 7 additional tests (test_xxx_yyy_meta, same root cause for all failures).

### Motivation
Tests should always work on master.

### Benefits
Obvious.

### Possible Drawbacks
None.

I've fixed the test_xxx_yyy_meta failures but still looking into test_storage_ignores_duplicate_ids:

```plain
______________________ test_storage_ignores_duplicate_ids ______________________

    def test_storage_ignores_duplicate_ids():
        storage.DATABASE = get_database(True)
        event = EiffelCompositionDefinedEvent()
    
        assert storage.insert_to_db(event, None)
>       assert not storage.insert_to_db(event, None)
E       assert not True
E        +  where True = <function insert_to_db at 0x7fc70b090598>(<eiffellib.events.eiffel_composition_defined_event.EiffelCompositionDefinedEvent object at 0x7fc70aa12c18>, None)
E        +    where <function insert_to_db at 0x7fc70b090598> = storage.insert_to_db

tests/test_storage.py:40: AssertionError
------------------------------ Captured log call -------------------------------
%: Event already exists in the database, skipping: {'data': {}, 'meta': {'id': 'b700ae67-cb5d-4000-b47b-5425fd94fab0', 'version': '3.0.0', 'type': 'EiffelCompositionDefinedEvent', 'time': 1599640132597}, 'links': []}
```

--- a/src/eiffel_graphql_api/graphql/db/database.py
+++ b/src/eiffel_graphql_api/graphql/db/database.py
@@ -74,7 +74,7 @@ def get_database(mock=False):
     :param mock: Whether the server should be mocked or not. Used for tests.
     :type mock: bool
     :return: Database instance.
-    :rtype: :obj:`pymongo.MongoClient.collection`
+    :rtype: :obj:`pymongo.database.Database`
     """
     if DATABASE is None:
         connect(mock)

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -18,8 +18,18 @@ import os
 import threading
 import pytest
 from eiffel_graphql_api.graphql.api import APP
+from eiffel_graphql_api.graphql.db.database import get_database
 from eiffel_graphql_api.graphql.db.database import get_client
 
+
+@pytest.fixture
+def mock_mongo():
+    """Inject a MongoDB client connected to a mock server with no database
+    in place, guaranteeing that we'll start from a clean slate.
+    """
+    client = get_client(mock=True)
+    client.drop_database(os.getenv("DATABASE_NAME"))
+    yield get_database(mock=True)
 
 
 def start():

--- a/tests/test_db/test_database.py
+++ b/tests/test_db/test_database.py
@@ -17,24 +17,23 @@
 
 from eiffellib.events.eiffel_composition_defined_event import EiffelCompositionDefinedEvent
 
-from eiffel_graphql_api import storage
-from eiffel_graphql_api.graphql.db.database import get_database
+from eiffel_graphql_api.graphql.db import database
 
 
-def test_storage_insert_single_doc():
-    storage.DATABASE = get_database(True)
+def test_insert_single_doc_inserts_doc(mock_mongo):
     event = EiffelCompositionDefinedEvent()
 
-    assert storage.insert_to_db(event, None)
-    collection = storage.DATABASE[event.meta.type]
-    result = collection.find_one({"_id": event.meta.event_id})
+    assert database.insert_to_db(event, None)
+    result = mock_mongo[event.meta.type].find_one({"_id": event.meta.event_id})
     del result["_id"]
     assert result == event.json
 
 
-def test_storage_ignores_duplicate_ids():
-    storage.DATABASE = get_database(True)
+def test_insert_single_doc_ignores_duplicate_ids(mock_mongo):
     event = EiffelCompositionDefinedEvent()
+    collection = mock_mongo[event.meta.type]
 
-    assert storage.insert_to_db(event, None)
-    assert not storage.insert_to_db(event, None)
+    assert database.insert_to_db(event, None)
+    assert collection.count_documents({}) == 1
+    assert database.insert_to_db(event, None)
+    assert collection.count_documents({}) == 1

--- a/tests/test_events/activity_canceled/test_activity_canceled.py
+++ b/tests/test_events/activity_canceled/test_activity_canceled.py
@@ -118,4 +118,4 @@ class TestActivityCanceled(TestCase):
         self.assertEqual(meta.get("version"), "3.0.0")
         self.assertEqual(meta.get("type"), "EiffelActivityCanceledEvent")
         self.assertEqual(meta.get("id"), "6a1abb6e-2c13-4a82-8fe2-012f8fe7c541")
-        self.assertEqual(meta.get("time"), "2019-12-10T08:32:35.892000")
+        self.assertEqual(meta.get("time"), 1575963155892)

--- a/tests/test_events/activity_finished/test_activity_finished.py
+++ b/tests/test_events/activity_finished/test_activity_finished.py
@@ -157,4 +157,4 @@ class TestActivityFinished(TestCase):
         self.assertEqual(meta.get("version"), "3.0.0")
         self.assertEqual(meta.get("type"), "EiffelActivityFinishedEvent")
         self.assertEqual(meta.get("id"), "bc73b474-4f5c-4931-b7d5-8588d0a6534a")
-        self.assertEqual(meta.get("time"), "2019-12-10T09:09:47.254000")
+        self.assertEqual(meta.get("time"), 1575965387254)

--- a/tests/test_events/activity_triggered/test_activity_triggered.py
+++ b/tests/test_events/activity_triggered/test_activity_triggered.py
@@ -128,7 +128,7 @@ class TestActivityTriggered(TestCase):
         self.assertEqual(meta.get("version"), "3.0.0")
         self.assertEqual(meta.get("type"), "EiffelActivityTriggeredEvent")
         self.assertEqual(meta.get("id"), "6a1abb6e-2c13-4a82-8fe2-012f8fe7c541")
-        self.assertEqual(meta.get("time"), "2019-12-09T13:43:57.093000")
+        self.assertEqual(meta.get("time"), 1575895437093)
 
     def test_activity_triggered_links(self):
         """Test that it is possible to query 'links' from activity triggered.

--- a/tests/test_events/announcement_published/test_announcement_published.py
+++ b/tests/test_events/announcement_published/test_announcement_published.py
@@ -135,4 +135,4 @@ class TestAnnouncementPublished(TestCase):
         self.assertEqual(meta.get("version"), "3.0.0")
         self.assertEqual(meta.get("type"), "EiffelAnnouncementPublishedEvent")
         self.assertEqual(meta.get("id"), "4baf56e6-404a-4132-a28b-9ed782f26293")
-        self.assertEqual(meta.get("time"), "2019-12-10T09:34:25.708000")
+        self.assertEqual(meta.get("time"), 1575966865708)

--- a/tests/test_events/artifact_created/test_artifact_created.py
+++ b/tests/test_events/artifact_created/test_artifact_created.py
@@ -210,4 +210,4 @@ class TestArtifactCreated(TestCase):
         self.assertEqual(meta.get("version"), "3.0.0")
         self.assertEqual(meta.get("type"), "EiffelArtifactCreatedEvent")
         self.assertEqual(meta.get("id"), "730f8573-cd69-41f5-81ad-d85aebf28d03")
-        self.assertEqual(meta.get("time"), "2019-12-10T09:57:08.603000")
+        self.assertEqual(meta.get("time"), 1575968228603)

--- a/tests/test_events/artifact_published/test_artifact_published.py
+++ b/tests/test_events/artifact_published/test_artifact_published.py
@@ -129,4 +129,4 @@ class TestArtifactPublished(TestCase):
         self.assertEqual(meta.get("version"), "3.0.0")
         self.assertEqual(meta.get("type"), "EiffelArtifactPublishedEvent")
         self.assertEqual(meta.get("id"), "031c2f9a-92f0-4cac-9320-e0113adafd7d")
-        self.assertEqual(meta.get("time"), "2019-12-10T13:34:15.471000")
+        self.assertEqual(meta.get("time"), 1575981255471)

--- a/tests/test_events/artifact_reused/test_artifact_reused.py
+++ b/tests/test_events/artifact_reused/test_artifact_reused.py
@@ -136,4 +136,4 @@ class TestArtifactReused(TestCase):
         self.assertEqual(meta.get("version"), "3.0.0")
         self.assertEqual(meta.get("type"), "EiffelArtifactReusedEvent")
         self.assertEqual(meta.get("id"), "0284875e-4f2f-4589-b6df-797afb039b88")
-        self.assertEqual(meta.get("time"), "2019-12-10T14:38:34.977000")
+        self.assertEqual(meta.get("time"), 1575985114977)

